pulp_rpm/tests/functional/api/test_crud_content_unit.py

```diff
@@ -158,3 +159,22 @@ def test_06_second_unit_raises_error(self):
         )
 
         self.assertEqual(task["error"]["description"], error_message)
+
+    def test_07_removing_content(self):
+        """
+        Test if content units support remove.
+
+        If bindings doesn't support 'delete',
```

Comment:
i am not super convinced jthis the best approach just because since bindgins is another layer after API you don't really know whether the bug is in api or bindings.
I would advocate to make a direct api call in this case.